blogs.json

```diff
@@ -1802,6 +1802,13 @@
             "site_url": "http://sealedabstract.com",
             "feed_url": "http://sealedabstract.com/feed/"
           },
+          {
+            "title": "Sebastian Osiński's Blog",
+            "author": "Sebastian Osiński",
+            "site_url": "https://osinski.dev",
+            "feed_url": "https://osinski.dev/index.xml",
+            "twitter_url": "https://twitter.com/seb_osinski"
+          }
```

Comment:
Think there is a comma `,` missing. Thanks.

Reply:
You're right, good catch! Thanks